modules/dnn/misc/python/test/test_dnn.py

```diff
@@ -286,38 +286,43 @@ def test_textdetection_model(self):
 
 
     def test_face_detection(self):
-        proto = self.find_dnn_file('dnn/opencv_face_detector.prototxt')
-        model = self.find_dnn_file('dnn/opencv_face_detector.caffemodel', required=False)
-        if proto is None or model is None:
-            raise unittest.SkipTest("Missing DNN test files (dnn/opencv_face_detector.{prototxt/caffemodel}). Verify OPENCV_DNN_TEST_DATA_PATH configuration parameter.")
+        model = self.find_dnn_file('dnn/onnx/models/yunet-202303.onnx', required=False)
+        if model is None:
+            raise unittest.SkipTest("Missing DNN test files (dnn/onnx/models/yunet-202303.onnx). Verify OPENCV_DNN_TEST_DATA_PATH configuration parameter.")
```

Comment:
if branch with exception is not needed. `find_dnn_file` uses `find_file` internally. The last one already implements the same logic: https://github.com/opencv/opencv/blob/bbe86e6deaea2ad80b14983a4abf68dabf85ec95/modules/python/test/tests_common.py#L30